app/web/features/profile/view/Badges.tsx

```diff
@@ -0,0 +1,45 @@
+import { Chip, Tooltip } from "@material-ui/core";
+import { User } from "proto/api_pb";
+import makeStyles from "utils/makeStyles";
+
+import { useBadges } from "../hooks/useBadges";
+
+interface Props {
+  user: User.AsObject;
+}
+
+const useStyles = makeStyles((theme) => ({
+  badgeContainer: {
+    marginTop: theme.spacing(1),
+  },
+  badge: {
+    marginInlineStart: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+}));
+
+export const Badges = ({ user }: Props) => {
+  const classes = useStyles();
+  const { badges } = useBadges();
+
+  if (badges === undefined) {
```

Comment:
This should probably also check for the `badgesList` being empty so it renders nothing (the fragment, `<></>`, or better with `null`)? As at the moment, it would return an empty div if user has no badges (mostly "ok" but may mess up styling sometimes unexpected further down the line)

Reply:
Is the PR otherwise roughly good?

Reply:
Yeah the rest looks fine to me!